src/flip/animateUnflippedElements/index.ts

```diff
@@ -85,6 +85,7 @@ const animateUnflippedElements = ({
         }
       } catch (DOMException) {
         // the element is already gone
+        closureResolve()
       }
     }
```

Comment:
This is definitely a bug, thanks for catching. It seems to me that this should instead be: 

```js
      try {
        parent.removeChild(element)
      } catch (DOMException) {
        // the element is already gone
      }
        exitingElementCount -= 1
        if (exitingElementCount === 0) {
          closureResolve()
      }
```

In case of an edge case where some exiting elements have been removed but not all of them, you'd still want to wait for them to leave before calling `closureResolve`. 
Does that make sense? (I'll look at this more closely tonight.)

Reply:
Yea that makes total sense, maybe also wrap that piece in a finally block?